coverage.txt

```diff
@@ -5,13 +5,13 @@ testing/test_all.py                   10      0   100%
 testing/yuca/__init__.py               0      0   100%
 testing/yuca/test_activations.py      36      1    97%
 testing/yuca/test_multiverse.py       86     23    73%
-testing/yuca/test_utils.py            23      1    96%
+testing/yuca/test_utils.py           166      1    99%
 yuca/__init__.py                       0      0   100%
 yuca/activations.py                  197    105    47%
 yuca/configs.py                       89     82     8%
 yuca/kernels.py                      118     99    16%
 yuca/multiverse.py                   546    259    53%
```

Comment:
Next issue will be to improve line coverage >= 75%, focusing on the 259 uncovered lines in multiverse here.